src/plugins/intel_cpu/src/transformations/cpu_opset/x64/pass/mlp_fusion.cpp

```diff
@@ -24,100 +24,118 @@
 #include "openvino/op/variadic_split.hpp"
 #include "openvino/pass/matcher_pass.hpp"
 #include "openvino/pass/pattern/matcher.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
 #include "openvino/util/pp.hpp"
 #include "transformations/cpu_opset/x64/op/llm_mlp.hpp"
-#include "transformations/utils/gen_pattern.hpp"
+#include "transformations/symbolic_transformations/symbolic_optimizations.hpp"
 
-using namespace ov::gen_pattern;
 using namespace ov::pass;
 
-ov::intel_cpu::MLPFusion::MLPFusion() {
-    MATCHER_SCOPE(MLPFusion);
+ov::intel_cpu::MLPFusion::MLPFusion() {}
 
-    auto input = makePattern("[?,?,?]");
+bool ov::intel_cpu::MLPFusion::run_on_model(const std::shared_ptr<ov::Model>& model) {
+    RUN_ON_MODEL_SCOPE(MLPFusion);
 
-    auto gate_proj_weight_compressed = makePattern<op::v0::Constant>({});  // [up_size, down_size]
-    auto gate_proj_weight = makePattern<op::v0::Convert>({gate_proj_weight_compressed}, {{"destination_type", "f32"}});
+    pass::SymbolicOptimizations symbolic_optimizations(false, get_pass_config());
+    auto symbolic_ctx_manager = symbolic_optimizations.get_manager();
 
-    auto up_proj_weight_compressed = makePattern<op::v0::Constant>({});  // [up_size, down_size]
-    auto up_proj_weight = makePattern<op::v0::Convert>({up_proj_weight_compressed}, {{"destination_type", "f32"}});
+    symbolic_ctx_manager->register_pass<intel_cpu::MLPFusionPass>();
 
-    auto down_proj_weight_compressed = makePattern<op::v0::Constant>({});  // [down_size, up_size]
-    auto down_proj_weight = makePattern<op::v0::Convert>({down_proj_weight_compressed}, {{"destination_type", "f32"}});
+    return symbolic_optimizations.run_on_model(model);
+}
+
+ov::intel_cpu::MLPFusionPass::MLPFusionPass() {
+    MATCHER_SCOPE(MLPFusionPass);
+
```

Comment:
```suggestion
    using ov::opv0::Constant, ov::op::v0::Convert, ...
```
Optional to consider:
Add using then all wrap_type etc. will be shorted, 
`pattern::wrap_type<Constant>`
or

`std::make_shared<Constant>`

Skip using for operators if same operator used but with different version

Reply:
it makes since, fixed